src/js/components/tasks/table.tsx

```diff
@@ -277,7 +274,6 @@ AssigneeTableCell.displayName = DataTableCell.displayName;
 
 const TaskTable = ({
   projectId,
```

Comment:
To support pagination for the new table, we'll need to adjust the `fetchMoreTasks` fn to account for `projectId` being missing, and probably also to accept a new `assignedToMe` boolean to filter by?